### PR TITLE
LRAC-14130 [F] Analyze test scenarios and check if they are possible to be covered on FE unit tests

### PR DESCRIPTION
**Pull request by fmarcio**
Jira issue https://liferay.atlassian.net/browse/LRAC-14130

### CODE REVIEW

**Comment by fmarcio** [comment]
ci:test:sf

**Comment by fmarcio** [comment]
ci:test:relevant

**Comment by fmarcio** [comment]
ci:test:relevant

**Comment by ViniLopes20** [comment]
ci:test:stable

**Comment by fmarcio** [comment]
ci:test:relevant

File: modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/settings/pages/__tests__/UsageOverview.jsx [hunk 1]
```diff
@@ -74,6 +74,68 @@ describe('UsageOverview', () => {
...
+				)
+			})
+		);
+
+		const {getByText} = render(
+			<WrappedComponent {...defaultProps} project={mockProject} />
+		);
+
+		expect(getByText('1% since July 8, 2018')).toBeInTheDocument();
+	});
```
Comment (interaminense) [inline modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/settings/pages/__tests__/UsageOverview.jsx:113]: Hi @fmarcio, could you group these two unit test?

**Review by interaminense — CHANGES_REQUESTED** [review]

File: modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/settings/pages/__tests__/UsageOverview.jsx (hunk 1 above)
Comment (fmarcio) [reply modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/settings/pages/__tests__/UsageOverview.jsx]: @interaminense done!

File: modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/settings/pages/__tests__/UsageOverview.jsx [hunk 2]
```diff
@@ -74,6 +74,68 @@ describe('UsageOverview', () => {
...
+		);
+
+		expect(getByText('105,000')).toBeInTheDocument();
+
+		expect(getByText('105,000')).toHaveClass('metric-limit');
+
+		expect(
+			getByText('Enterprise Plan 95,000 + 5,000 Add-On (2x)')
+		).toBeInTheDocument();
+	});
```
[the remark this answers is not in the document]
Comment (fmarcio) [reply modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/settings/pages/__tests__/UsageOverview.jsx]: @interaminense done!

**Comment by fmarcio** [comment]
ci:test:sf

**Comment by fmarcio** [comment]
ci:test:relevant

**Comment by ViniLopes20** [comment]
ci:test:stable

**Review by interaminense — APPROVED** [review]
LGTM!

**Comment by interaminense** [comment]
ci:forward

**Comment by fmarcio** [comment]
ci:forward

**Comment by interaminense** [comment]
ci:forward:force

**Pull request closed without merging**